src/lava/lib/peripherals/liveplot/liveplot.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import cv2
+import numpy as np
+
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.ports.ports import InPort
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.model.py.ports import PyInPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.resources import CPU
+from lava.magma.core.decorator import implements, requires
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+
+import time
+
+
+class OpenCVDisplay(AbstractProcess):
+    """Process that receives datas from connected processmodel.
+    The frame data which are needed to display from connected processmodels
+    need a corresponding InPort which is connected to an OutPort.
+    The number of input ports is decided by the number of
```

Comment:
I do not quite understand, there will always be only 1 InPort in the current implementation?

Reply:
Nope. Users could add InPorts if need. These words just show that if users want to show multiple frames, they need to add OutPort from other Processes and connect them to the InPorts in this liveplot Process.